src/components/logo/index.test.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { RootState } from '../../store';
+
+import { Link } from 'react-router-dom';
+import { Apps } from '../../lib/apps';
+import { Container } from '.';
+
+describe('Logo', () => {
+  const subject = (props: any = {}) => {
+    const allProps = {
+      calssName: '',
+      type: '',
+      ...props,
+    };
+
+    return shallow(<Container {...allProps} />);
+  };
+
+  it('adds className', () => {
```

Comment:
looks good, but let's add a test to make sure the link is getting the right `to`, please. :)